src/server/prepare.js

```diff
@@ -20,7 +20,16 @@ function prepare() {
 
         lastPlatform = platform;
 
-        exec('cordova prepare ' + platform, function (err, stdout, stderr) {
+        var projectRoot;
+        try {
+            projectRoot = config.projectRoot;
+        } catch (error) {
+            projectRoot = process.cwd();
```

Comment:
We expect `config.projectRoot` to be set. If it isn't, that is an error situation and we shouldn't proceed. Perhaps we should call `d.reject(error);` here instead (it will be handled by the top level error handler, which will display the error to the user, then bail)?